packages/migration/ecosysystem.go

```diff
@@ -1136,7 +1136,11 @@ If("#key_id#" == EcosysParam("founder_account")){
 		}
 		action {
 			DBUpdate("keys", $key_id,"-amount", $amount)
-			DBUpdate("keys", $recipient,"+amount", $amount)
+			if DBFind("keys").Columns("id").WhereId($recipient).One("id") == nil {
```

Comment:
all db requests should be in models pacakge

Reply:
This is Symvolio. There is not access to models from Symvolio.